nengo/networks/workingmemory.py

```diff
@@ -5,15 +5,16 @@
 
 
 def InputGatedMemory(n_neurons, dimensions, fdbk_scale=1.0, gate_gain=10,
-                     difference_gain=1.0, reset_gain=3,
-                     mem_config=None, net=None):
+                     difference_gain=1.0, reset_gain=3, recurrent_synapse=0.1,
+                     difference_synapse=None, net=None):
     """Stores a given vector in memory, with input controlled by a gate."""
     if net is None:
         net = nengo.Network(label="Input Gated Memory")
 
-    if mem_config is None:
-        mem_config = nengo.Config(nengo.Connection)
-        mem_config[nengo.Connection].synapse = nengo.Lowpass(0.1)
+    if difference_synapse is None:
+        difference_synapse = recurrent_synapse
+    mem_config = nengo.Config(nengo.Connection)
+    mem_config[nengo.Connection].synapse = nengo.Lowpass(recurrent_synapse)
```

Comment:
I don't think this should be done this way.  Get rid of the ```Config``` object completely and just do

```python
        nengo.Connection(net.mem.output, net.mem.input,
                              transform=fdbk_scale, synapse=recurrent_synapse)
```